CHANGELOG.md

```diff
@@ -1,5 +1,73 @@
 # Changelog
 
+## [v1.12.2](https://github.com/mongodb/terraform-provider-mongodbatlas/tree/v1.12.2) (October 17, 2023)
```

Comment:
I believe this is not the format @Zuhairahmed likes, we might want to change date-format=%B %d, %Y.

Reply:
updated